deploy/eck-stack/examples/elasticsearch/quickstart.yaml

```diff
@@ -0,0 +1,84 @@
+---
+eck-elasticsearch:
+  # Name of the Elasticsearch instance.
+  #
+  name: "quickstart"
+
+  # Version of Elasticsearch.
+  #
+  version: 8.2.3
+
+  # Labels that will be applied to Elasticsearch.
+  #
+  labels: {}
+
+  # Annotations that will be applied to Elasticsearch.
+  #
+  annotations: {}
+
+  # Settings for configuring stack monitoring.
+  # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-stack-monitoring.html
+  #
+  monitoring: {}
+
+  # Control the Elasticsearch transport module used for internal communication between nodes.
+  # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-transport-settings.html
+  #
+  transport: {}
+
+  # Settings to control how Elasticsearch will be accessed.
+  # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-accessing-elastic-services.html
+  #
+  http: {}
+
+  # Control Elasticsearch Secure Settings.
+  # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-es-secure-settings.html#k8s-es-secure-settings
+  #
+  secureSettings: {}
+
+  # Settings for limiting the number of simultaneous changes to an Elasticsearch cluster.
+  # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-update-strategy.html
+  #
+  # The default are the default settings for ECK.
+  #
+  updateStrategy: {}
+
+  # Controlling of connectivity between remote clusters within the same kubernetes cluster.
+  # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-remote-clusters.html
+  #
+  remoteClusters: {}
+
+  # Node configuration settings.
+  # The node roles which can be configured here are:
+  # - "master"
+  # - "data_hot"
+  # - "data_cold"
+  # - "data_frozen"
+  # - "data_content"
+  # - "ml"
+  # - "ingest"
+  # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-node-configuration.html
+  #
+  nodeSets:
+  - name: default
+    count: 1
+    config:
+      node.store.allow_mmap: false
+    podTemplate:
+      spec:
+        containers:
+        - name: elasticsearch
+          resources:
+            requests:
+              cpu: 1
+            limits:
+              memory: 1Gi
+  # Uncomment these lines in case you want to increase vm.max_map_count and/or use initContainers.
+  # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-virtual-memory.html
+  #
+  #        initContainers: # 
+  #        - name: sysctl
+  #          securityContext:
+  #            privileged: true
+  #            runAsUser: 0
+  #          command: ['sh', '-c', 'sysctl -w vm.max_map_count=262144']
```

Comment:
This seems to duplicate more or less the `values.yaml` file of the chart. Do we need that? Also should the examples be placed in the corresponding charts?